explore/src/main/scala/explore/targets/TargetColumns.scala

```diff
@@ -109,7 +117,42 @@ object TargetColumns:
         baseColumn(NameColumnId, Target.name.get)
           .setCell(_.value.map(_.toString).orEmpty)
           .setSize(120.toPx)
-          .sortableBy(_.toString)
+          .sortableBy(_.map(_.toString))
+      )
+
+    val catalogColumns =
+      List(
+        baseColumn(
+          CatalogName,
+          Target.catalogInfo.andThen(some).andThen(CatalogInfo.catalog).getOption
+        )
+          .setCell(_.value.flatten.map(_.shortName).orEmpty)
+          .setSize(100.toPx)
+          .sortableBy(_.flatten.map(_.toString)),
```

Comment:
is this the same as `foldMap(_.toString)`

Reply:
Not exactly. The input parameter is an `Option[Option[CatalogName]]`